exercise_flexbox/index.html

```diff
@@ -0,0 +1,93 @@
+    <header>
+        <h2>The VC Experience</h2>
+        <nav class="navbar">
+            <p>
+                <a href="#">Mission Statement</a>
+            </p>
+            <p>
+                <a href="#">Portfolio</a>
+            </p>
+            <p>
+                <a href="#">Investment Team</a>
+            </p>
+        </nav>
+    </header>
+    <main>
+        <section class="objective">
+            <div class="objective-content">
+                <h3>The VC Experience </h3>
+                <h5>Our fund is committed to the furthering entrepeneurs in the mobility sector.</h5>
+            </div>
+        </section>
+        <section class="portfolio">
+            <div class="portfolio-content">
+                <h3>Our Mobility Portfolio</h3>
+                <h5>Impacting diverse sectors of mobility to better the human experience and the world we inhabit</h5>
+                <div class="portfolio-companies">
+                    <div class="company">
+                        <img src="./resources/images/hangar.jpg"/>
+                        <span>AeroArgonauts</span>
+                    </div>
+                    <div class="company">
+                        <img src="./resources/images/dashboard.jpg"/>
+                        <span>AutoBueno</span>
+                    </div>
+                    <div class="company">
+                        <img src="./resources/images/blurred-lights.jpg"/>
+                        <span>LaneChatter</span>
+                    </div>
+                    <div class="company">
+                        <img src="./resources/images/tires.jpg"/>
+                        <span>TigerTreads</span>
+                    </div>
+                    <div class="company">
+                        <img src="./resources/images/truck.jpg"/>
+                        <span>ValetToday</span>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="team-container">
+            <h3>Our Team</h3>
+            <div class="team">
+                <div class="member">
+                    <img src="./resources/images/employee-one.jpg"/>
+                    <h5>Lisa Fischer</h5>
+                    <h6>C.E.O. - Founder and Principal Investor</h6>
+                </div>
+                <div class="member">
+                    <img src="./resources/images/employee-two.jpg"/>
+                    <h5>Alex Lasker</h5>
+                    <h6>Partner - Research & Development</h6>
+                </div>
+                <div class="member">
+                    <img src="./resources/images/employee-three.jpg"/>
+                    <h5>Omar Carlsen</h5>
+                    <h6>Partner - Partnerships & Marketing</h6>
+                </div>
+            </div>
+        </section>
+    </main>
+    <footer>
+        <section class="contact">
+            <h3>The VC Experience</h3>
+            <h5><em>introduction@vcexperience.investors</em></h5>
```

Comment:
You coul use an `a` tag to send mail directly to this direction using

`href="mailto:introduction@vcexperience.investors"`